gulpfile.js

```diff
@@ -113,16 +109,16 @@ gulp.task('istanbul-pre-test', function () {
 });
 
 gulp.task('test-cov', ['istanbul-pre-test'], function () {
-  return gulp.src(['test/*.js', 'test/support/*.js'])
```

Comment:
This glob causes `test-cov` to fail because it includes a failing test binary-fallback ( #459 )